modules/BuildTools.Tasks/UnzipArchive.cs

```diff
@@ -62,7 +67,18 @@ public override bool Execute()
             {
                 foreach (var entry in zip.Entries)
                 {
-                    var fileDest = Path.Combine(Destination, entry.FullName);
+                    var entryPath = entry.FullName;
+                    if (!DisablePathNormalization)
+                    {
+                        if (entry.FullName.IndexOf('\\') >= 0)
```

Comment:
Should these be `Path.DirectorySeperator` or is there some reason that it really always should be '/'?

Reply:
`\` is considered a valid file name character on \*nix file systems, which is why we end up with unzip layouts like this. 
![image](https://user-images.githubusercontent.com/2696087/34689348-057fcd38-f46a-11e7-8542-f39aad951790.png)

We should never produce ZIPs with backslashes in the entry name. `ZipArchive` normalizes these. I've hard-coded the check this way because I want our builds to issue warnings about bad zips.